networking/src/Models/IQuestion.ts

```diff
@@ -17,7 +20,28 @@ export interface IQuestion {
 }
 
 export interface IChoice {
+    id?: string
     text: string
     reason?: string
     isAnswer: boolean
+}
+
+export interface IAnswerSettings {
+    answerType: string
+    answerPrecision: string
+}
+
+export interface IResponse {
+    rawAnswer: string
+    normAnswer: string[]
+    isCorrect: boolean
+    isSelectedMistake: boolean
+    count: number
+    teams: Array<IResponseTeam>
+}
+
+export interface IResponseTeam {
+    team: string
+    id: string
+    confidence: ConfidenceLevel
```

Comment:
`IResponse` and `IResponseTeam` are objects that are used to populate the graphs on `host` per the screenshot below:

<img width="545" alt="Screenshot 2024-01-09 at 13 49 22" src="https://github.com/rightoneducation/righton-app/assets/79417944/28d9b2c9-e562-4705-8b79-ce9265328d5a">

A lot of this data requires some level of processing and I only want to do that once. Therefore, I process the data and then store it under the question object. Then, if I ever need to retrieve it, I don't need to do a bunch of maps through the team, teammember and teamanswer objects but can just go into the question and grab it. I'm duplicating data here but I think it's a bit easier to retrieve. 

Additionally, I'm looking toward the Analytic Dashboard that we'll be working on this year and I think it will be useful to store this kind of data under the questions for easy retrieval as well. 

Definitely let me know if you think this is confusing or offbase though.

Reply:
hmm, if we're saving it under `IQuestion`, we might wanna look into storing it as a `Map`, then you won't need `count` and we can probably use the `normAnswer` as the key for the map and the value can be array of all responses.